<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1328</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BloodCellApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="spacing">
     <number>3</number>
    </property>
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Tools</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QPushButton" name="bLearnModel">
         <property name="text">
          <string>Learn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bTest">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bDevTest">
         <property name="text">
          <string>DevTest</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QListView" name="lvImages">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTabWidget" name="tabWidget">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <widget class="QWidget" name="tRecognition">
            <attribute name="title">
             <string>Recognition</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QTableView" name="tvAtlas">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>350</height>
                </size>
               </property>
               <property name="iconSize">
                <size>
                 <width>50</width>
                 <height>50</height>
                </size>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>150</number>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>150</number>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="verticalHeaderDefaultSectionSize">
                <number>55</number>
               </attribute>
               <attribute name="verticalHeaderMinimumSectionSize">
                <number>55</number>
               </attribute>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>281</width>
                 <height>55</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tConfiguration">
            <attribute name="title">
             <string>Configuration</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QComboBox" name="cbConfigType">
               <item>
                <property name="text">
                 <string>Dimensions</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Classes</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QStackedWidget" name="swConfigData">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="pDimensions">
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QPushButton" name="bMarkSize">
                     <property name="text">
                      <string>Mark an object of known size:</string>
                     </property>
                     <property name="checkable">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="eKnownSizeUnit">
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>0</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="comboBox">
                     <item>
                      <property name="text">
                       <string>m</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>cm</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>mm</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Âµm</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>nm</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_4">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QLabel" name="lRbcSize">
                     <property name="text">
                      <string>=</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="eKnownSizePx">
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="readOnly">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Pixel</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>230</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QPushButton" name="bSaveRBCConfig">
                     <property name="text">
                      <string>Save</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="bResetRBCConfig">
                     <property name="text">
                      <string>Reset</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="pClasses">
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_8">
                   <item>
                    <widget class="QTableView" name="tvClasses">
                     <property name="selectionMode">
                      <enum>QAbstractItemView::SingleSelection</enum>
                     </property>
                     <property name="selectionBehavior">
                      <enum>QAbstractItemView::SelectRows</enum>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>50</width>
                       <height>50</height>
                      </size>
                     </property>
                     <attribute name="horizontalHeaderVisible">
                      <bool>false</bool>
                     </attribute>
                     <attribute name="horizontalHeaderStretchLastSection">
                      <bool>true</bool>
                     </attribute>
                     <attribute name="verticalHeaderVisible">
                      <bool>false</bool>
                     </attribute>
                     <attribute name="verticalHeaderDefaultSectionSize">
                      <number>50</number>
                     </attribute>
                     <attribute name="verticalHeaderMinimumSectionSize">
                      <number>50</number>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_9">
                     <item>
                      <widget class="QPushButton" name="bAddClass">
                       <property name="text">
                        <string>+</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="bRemoveClass">
                       <property name="text">
                        <string>-</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="bGetIcon">
                       <property name="text">
                        <string>Icon</string>
                       </property>
                       <property name="checkable">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_6">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <widget class="QListView" name="lvModelImages">
                     <property name="iconSize">
                      <size>
                       <width>80</width>
                       <height>60</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_8">
                     <item>
                      <widget class="QPushButton" name="bRemoveModelInstance">
                       <property name="text">
                        <string>-</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_7">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <spacer name="verticalSpacer_3">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <widget class="QPushButton" name="bSuggest">
                       <property name="text">
                        <string>Suggest</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="bAcceptAllSuggested">
                       <property name="text">
                        <string>Accept all</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="bRejectAllSuggested">
                       <property name="text">
                        <string>Reject all</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_5">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbImage">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="title">
         <string>Image</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>3</number>
         </property>
         <property name="rightMargin">
          <number>3</number>
         </property>
         <property name="bottomMargin">
          <number>3</number>
         </property>
         <item>
          <widget class="QGraphicsView" name="gvImage"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1328</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionLoad"/>
    <addaction name="actionOpen_file"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClear_images"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load directory</string>
   </property>
  </action>
  <action name="actionClear_images">
   <property name="text">
    <string>Clear images</string>
   </property>
  </action>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="text">
    <string>New Project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
